action.yml

```diff
@@ -0,0 +1,82 @@
+name: Configure AWS Profile
+
+description: Configures an OIDC AWS Role Session as an aws config profile.
+
+
+inputs:
+  role-to-assume:
+    description: "ARN of the role to assume."
+    required: true
+  aws-profile-name:
+    description: "Name of profile to be created. If not set ID of step is used."
+    required: false
+  role-duration-seconds:
+    description: "Role duration in seconds. For role chaining max: 3600."
+    default: "7200"
+    required: false
+  aws-region:
+    description: "Name of AWS region to be used."
+    default: "eu-west-1"
+    required: false
+  aws-access-key-id:
+    description: "AWS Access Key ID."
+    required: false
+  aws-secret-access-key:
+    description: "AWS Secret Access Key."
+    required: false
+  aws-session-token:
+    description: "AWS Session Token"
+    required: false
+  aws-source-profile:
+    description: "Name of source profile for assuming role by SDKs"
+    required: false
+
+outputs:
+  aws-profile:
```

Comment:
maybe is better use same like input `aws-profile-name`

Reply:
like discussed in previous pr, this naming is on purpose, so name for name which is not creatred yet, without name, created profile